specification/src/main/asciidoc/coreprofile/CoreProfileDefinition.adoc

```diff
@@ -17,14 +17,14 @@ The following technologies are required components of the Core Profile:
 
 === Optional Components
 
-There are no optional components in the Core Profile.
+The optional Jakarta XML Binding support in Jakarta RESTful Web Services implementations is not required for the Core Profile.
+
+Jakarta Expression Language support is not required in  Jakarta Contexts and Dependency Injection Lite implementations since this is part of the Full CDI specification.
```

Comment:
minor nit, replace double space after `required in` with single space.

Reply:
Fixed.